my-app/src/Components/Counter.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+class Counter extends React.PureComponent {
+  constructor() {
+    super();
+    this.increase = this.increase.bind(this);
```

Comment:
Not an error, but there is no need to bind function. You can use arrow function:
```js
increase = () => {
  this.setState(prevState => ({ counter: prevState.counter + 1 }));
}

decrease = () => {
  this.setState(prevState => ({ counter: prevState.counter - 1 }));
}
```

But if you decide to use `bind` you should bind `decrease` function:
```js
this.decrease = this.decrease.bind(this);
```